.drone.star

```diff
@@ -63,7 +63,7 @@ def localApiTestsOcStorage(ctx, coreBranch = 'master', coreCommit = ''):
           'TEST_EXTERNAL_USER_BACKENDS':'true',
           'REVA_LDAP_HOSTNAME':'ldap',
           'TEST_OCIS':'true',
-          'BEHAT_FILTER_TAGS': '~@skipOnOcis-OC-Storage',
+          'BEHAT_FILTER_TAGS': '~@skipOnOcis-OC-Storage&&~@preview-extension-required',
```

Comment:
Note: `~@preview-extension-required` tag filter should have been on the core API acceptance tests.

I moved it there in #426 and deleted the ``previews.feature` entries out of the expected failures file.